src/lib/content/courses/niftory/en/chapter1/lesson1.md

```diff
@@ -0,0 +1,260 @@
+
+Whether you are trying to build your own digital marketplace, or experimenting with completely new web3 use-cases we haven't even dreamt of, Niftory will work for you!
+
+## What problem does Niftory solve? 
+
+The API and platform is designed to be used by all developers, especially those who just want to integrate web3 content into their applications without the overhead of becoming blockchain engineers.
+We don’t require you to deploy your own smart contracts, and our API is a normal GraphQL endpoint—you can be a complete beginner in web3 and still launch a digital marketplace in minutes. 
+
+These are our core set of Capabilities and during this course, we'll go over each of them to see how they will help you build the DApp of your dreams: 
+
+1. **Data management**. Off-chain and on-chain data managed through a single API. We combine the reliability and persistence of the Flow blockchain with the speed and cost of traditional web2 data management.
+2. **User authentication**. Using OAuth 2.0, as well as wallet management and verification.
+3. **GraphQL API**. Easy way of querying, minting and transferring NFTs. 
+4. **Scalable**. All of this is built on scalable micro-service architecture and industrial grade data storage.
+
+## So, what IS Niftory? 
+
+Great question. Niftory is a set of tools to help you build, launch and maintain DApps. It consists of two different, interrelated pieces: 
+1. **The Niftory Web3 API**. The Niftory API is all you need to launch a new Web3 experience, without having to think about the blockchain or any complex infrastructure setup. 
+2. **The Admin panel**. The Admin Portal is where you deploy your smart contract, get your API keys, manage your team, design NFTs, and track how your app is doing as users interact with your NFTs.
+
+## Same functionality, different tools
+
+To see Niftory in action, it's useful to see it compared to the code you'd have to write if you were to develop your app exclusively on top of Flow (using the Cadence programming language). Here, we're going to show 
+
+Present sample code with and without Niftory to highlight the difference and simplicity that Niftory brings.
+
+### Using Cadence and Flow directly
+
+This code has been pulled from the [flow example](https://github.com/onflow/kitty-items/). 
+
+It consists of three parts: 
+1. Javascript Function to call the transfer
+2. Transfer in Cadence. 
+3. Flow Service (for signing, creating transactions, etc with FCL - Flow Client Library)
+
+You will also have to update the Front-end to invoke a button that calls the transfer but we've skipped it for this example since that would be the same between Niftory and Cadence. 
+
+1. Javascript To Call Transfer: 
```

Comment:
@jacob-tucker - one thing I'm worried about is this huge compare/contrast between code snippets showing how something would be done in cadence contrasted with how it should be done in Niftory. Any tips? Will this be somewhat expanded in the site?